variable.tf

```diff
@@ -0,0 +1,29 @@
+variable "region" {
+    description = "The aws region"
+    type = string
+    default = "us-west-1"
```

Comment:
remove all defaut values from here, use terrafrom.tfvars file to populate these values.

Reply:
content of file will look like
cidr_block: "10.0.0.0/24"
....